lib/msal-common/src/utils/Constants.ts

```diff
@@ -74,7 +74,8 @@ export enum PersistentCacheKeys {
     CLIENT_INFO = "client.info",
     ADAL_ID_TOKEN = "adal.idtoken",
     ERROR = "error",
-    ERROR_DESC = "error.description"
+    ERROR_DESC = "error.description",
+    ACTIVE_ACCOUNT = "active-account"
```

Comment:
nit: Everything else is separated by a `.`, why a `hyphen` for this name?

Reply:
Other cache entries use hyphen; `server-telemetry` for instance. I believe the `.` is legacy for migration from v1

Reply:
Okay